contents/blog/using-vc-carefully.md

```diff
@@ -0,0 +1,117 @@
+
+## What happened next - the PostHog dark ages / turns out board meetings are very important
+
+Things got a bit messy.
+
+We'd hired quite a bit. We didn't really have revenue but didn't really care about that, yet. We did have lots of users.
+
+At the same time:
+
+* customers wanted a paid version of the product (asking for extra features or for us to host the product for them)
+* tons of the world's best investors were trying to talk to us
+* my daughter got cancer (she's now doing well thankfully)
+
+I wanted to keep working, but it meant I had to look after my daughter. I felt that if I tried to sell _and_ fundraise, I'd do both badly.
+
+We figured it was an existential problem not to have revenue, and that it'd be easier to figure it out now instead of when we're bigger.
+
+I couldn't be more glad that's what we picked. Results:
+
+![A graph showing revenue over time for PostHog - it climbs exponentially](../images/blog/vc-as-tool/revenue.jpg)
+
+## How to start focusing on revenue in the first place
+
+This we nailed.
+
+First, when we launched the free open source product we made clear from the outset that we would have a paid offering. We did this by explaining our different offerings on our pricing page.
+
+Once we started getting regular demand for the paid product and after feeling that the open source product was building a big community and users were really getting value from it, we decided to focus on revenue.
+
+We quickly set the goal of '5 reference customers' for the entire organization. We tracked the things each potential customer had in common - very specific things like "do they have product market fit" / "do they have a data warehouse" / "have they used our free product before". Quickly we could spot the kind of deals that would close and the kind that wouldn't - it was really useful tracking both the deals that closed _and_ the ones that dragged on.
+
+There were three surprising lessons we had during this phase:
+
+1 - charging customers for a product led to more usage. Early paying customers were more demanding of quality. When we fixed quality issues, our free _and_ paid products both accelerated
+2 - how important it is to track your anti-Ideal Customer Profile. Don't just monitor deals that close, but track those that are dragging on - what do they have in common?
+3 - how quickly we went from getting the first 5 paying customers, to 100.
+
+## We have Product Market Fit, time to raise more money?
+
+Many companies do this. The reason is pretty simple - if you grow by hiring salespeople, you can use venture capital to hire faster than you could otherwise afford.
+
+We took the stance that we should get to $10M in recurring revenue, and then could go nuts. This would mean we could raise capital easily, on great terms, and it'd mean we had very solid product market fit.
```

Comment:
```suggestion
We took the stance that we should get to $10m in recurring revenue, and then (could) go nuts. This would mean we could raise capital easily, on great terms, and we'd have very solid product-market fit.
```